From cce3c65947475f8dd808b35c259c194592ac37f2 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 14 Jun 2012 12:20:44 +0800
Subject: [PATCH 009/281] ENGR00000005 Move the viv-gpu configure into
 BoardConfig

If put it into product makefile, the PREBUILT_FSL_IMX_GPU
will not be defined before including it.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 arm2_6dq/BoardConfig.mk    | 1 +
 imx6/arm2_6dq.mk           | 1 -
 imx6/sabreauto_6q.mk       | 2 --
 imx6/sabresd_6dq.mk        | 1 -
 sabresd_6dq/BoardConfig.mk | 1 +
 5 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/arm2_6dq/BoardConfig.mk b/arm2_6dq/BoardConfig.mk
index 8f766c6..ea05d1e 100755
--- a/arm2_6dq/BoardConfig.mk
+++ b/arm2_6dq/BoardConfig.mk
@@ -4,6 +4,7 @@
 
 include device/fsl/imx6/soc/imx6dq.mk
 include device/fsl/imx6/arm2/BoardConfigArm2.mk
+include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
 BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0
 TARGET_BOOTLOADER_CONFIG := 6q:mx6q_arm2_android_config 6dl:mx6dl_arm2_config
diff --git a/imx6/arm2_6dq.mk b/imx6/arm2_6dq.mk
index 4f4e7d0..a84204f 100644
--- a/imx6/arm2_6dq.mk
+++ b/imx6/arm2_6dq.mk
@@ -8,7 +8,6 @@ PRODUCT_NAME := arm2_6dq
 PRODUCT_DEVICE := arm2_6dq
 
 include device/fsl/imx6/arm2/ProductCommonArm2.mk
-include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
 PRODUCT_COPY_FILES +=	\
 	device/fsl/arm2_6dq/init.rc:root/init.freescale.rc
diff --git a/imx6/sabreauto_6q.mk b/imx6/sabreauto_6q.mk
index 3f76e24..39d4ee1 100644
--- a/imx6/sabreauto_6q.mk
+++ b/imx6/sabreauto_6q.mk
@@ -18,5 +18,3 @@ PRODUCT_COPY_FILES += \
 # GPU files
 
 DEVICE_PACKAGE_OVERLAYS := device/fsl/sabreauto_6q/overlay
-
-include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
diff --git a/imx6/sabresd_6dq.mk b/imx6/sabresd_6dq.mk
index e08c46f..6d0c567 100644
--- a/imx6/sabresd_6dq.mk
+++ b/imx6/sabresd_6dq.mk
@@ -8,4 +8,3 @@ PRODUCT_NAME := sabresd_6dq
 PRODUCT_DEVICE := sabresd_6dq
 
 include device/fsl/imx6/sabresd/SabreSDProductCommon.mk
-include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index 4dea943..0cd5f9c 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -4,6 +4,7 @@
 
 include device/fsl/imx6/soc/imx6dq.mk
 include device/fsl/imx6/sabresd/SabreSDBoardConfigComm.mk
+include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
 BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 fbmem=10M vmalloc=400M androidboot.console=ttymxc0
 TARGET_BOOTLOADER_CONFIG := 6q:mx6q_sabresd_android_config 6dl:mx6dl_sabresd_android_config
-- 
1.8.0

